R/utilities-writing-report.R

```diff
@@ -221,6 +221,16 @@ renderWordReport <- function(fileName, intro = NULL, createWordReport = FALSE, w
     # Issue #968 Subscript and superscript rendering
     lineContent <- gsub(pattern = "<sup>|</sup>", replacement = "^", lineContent)
     lineContent <- gsub(pattern = "<sub>|</sub>", replacement = "~", lineContent)
+    # Expose openxml translations for users
+    # Manual page break
+    lineContent <- gsub(pattern = "<pagebreak>", replacement = '<w:p><w:r><w:br w:type="page"/></w:r>`{=openxml}', lineContent)
+    # Alignment
+    lineContent <- gsub(pattern = "</div>", replacement = '', lineContent)
+    lineContent <- gsub(pattern = '<div align=\"right\">', replacement = '`<w:pPr><w:jc w:val="right"/></w:pPr>`{=openxml}', lineContent)
+    lineContent <- gsub(pattern = '<div align=\"left\">', replacement = '`<w:pPr><w:jc w:val="left"/></w:pPr>`{=openxml}', lineContent)
+    lineContent <- gsub(pattern = '<div align=\"center\">', replacement = '`<w:pPr><w:jc w:val="center"/></w:pPr>`{=openxml}', lineContent)
+    lineContent <- gsub(pattern = '<div align=\"justify\">', replacement = '`<w:pPr><w:jc w:val="both"/></w:pPr>`{=openxml}', lineContent)
+    
```

Comment:
@pchelle 
As far as I see, if I have something like `<div align="center"> Tralala </div>` it will be replaced by 
`<w:pPr><w:jc w:val="both"/></w:pPr>{=openxml} Tralala`
Does the open xml setting then apply for the rest of the line (and the next line is not affected)?

If I have a table with the div setting in every cell - will the openxml setting then be applied just to the cell?

|`<div align="center">` Header`</div>` | `<div align="center">`Header `</div>`|
|--------|--------|
| `<div align="left">`Cell `</div>`|`<div align="left">` Cell `</div>`|
| `<div align="left">`Cell`</div>` |`<div align="left">` Cell `</div>`|

Reply:
Yes, both inline and block openxml `<w:pPr><w:jc w:val="center"/></w:pPr>` only center the next line(s) as long as they are in the same paragraph.

```
|**C_max**                        |Number |Ratio [%] |
|:----------------------------|:------|:---------|
|Points total                 |`<w:pPr><w:jc w:val="right"/></w:pPr>`{=openxml} 2      |-        |
|Points within Guest *et al.* |`<w:pPr><w:jc w:val="center"/></w:pPr>`{=openxml} 0      |0         |
|Points within 2 fold         |`<w:pPr><w:jc w:val="left"/></w:pPr>`{=openxml}2      |100       |
```

converts into 

![image](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/assets/45343665/04730c59-fdd1-4a36-9151-e62785c7a34e)

Reply:
Cool! Didn't expect so much intelligence from openXML :)